package.json

```diff
@@ -32,6 +32,8 @@
     "ember-cli-babel": "^7.26.11",
     "ember-cli-htmlbars": "^6.0.1",
     "ember-cli-sass": "^11.0.1",
+    "ember-concurrency": "^2.2.1",
+    "ember-concurrency-decorators": "^2.0.3",
```

Comment:
This can actually go away. All the decorators are now support on the main `ember-concurrency` package.

Reply:
Do you mind of we push this refactoring to a follow-up PR? Would love to first land some test coverage before refactoring.